diff --git a/dist/view-transitions-polyfill.js b/dist/view-transitions-polyfill.js
index d746bd9391299603d55cd5bc93cb59fe9802c5e3..b29a558ed6d4218fffe92dd1d8e394494b8fd45d 100644
--- a/dist/view-transitions-polyfill.js
+++ b/dist/view-transitions-polyfill.js
@@ -8008,7 +8008,7 @@ const Je = class Je {
       if (s instanceof dt)
         return;
     }
-    this.captureElements("new"), await rm(), this.setupGroups(), this.newElements.forEach((s) => er(s, !1)), this.transitionPromises.ready.resolve(), this.phase = "animating";
+    this.captureElements("new"), this.setupGroups(), this.newElements.forEach((s) => er(s, !1)), this.transitionPromises.ready.resolve(), this.phase = "animating";
     const i = this.viewTransitionElement.getAnimations({
       subtree: !0
     }).map((s) => s.finished);
